parameters.cpp

```diff
@@ -374,6 +380,13 @@ bool Parameters::getParameters(){
       }
       return false;
    }
+   if ( P::systemWriteDistributionWriteShellStride.size() != P::systemWriteDistributionWriteShellRadius.size()) {
+      if(myRank == MASTER_RANK) {
+         cerr << "ERROR io.system_write_distribution_shell_stride should have the same size as " <<
```

Comment:
This is nitpicking but it would be clearer to the user if it said something like "You should set the same number of io.system_write_distribution_shell_stride and io.system_write_distribution_shell_radius".